gdx/src/com/badlogic/gdx/utils/GdxNativesLoader.java

```diff
@@ -18,6 +18,7 @@
 
 public class GdxNativesLoader {
 	static public boolean disableNativesLoading = false;
+	static public boolean disableLwjglNativesLoading = false;
```

Comment:
Can you explain again, when this is needed?
Also, wouldn't it make sense to move this to `LwjglNativesLoader`?
Note that `LwjglNativesLoader` already has a `load` field that you could use instead with the same effect I think.

Reply:
basically one GL application is already loding LWJGL (i.e. from different lib folders) before LIBGDX, so we just give the option to avoid a second unnecessary loading.